phongo_compat.h

```diff
@@ -91,6 +91,12 @@
 #define ADD_ASSOC_ZVAL(_zv, _key, _value) add_assoc_zval(_zv, _key, _value);
 #define ADD_ASSOC_NULL_EX(_zv, _key) add_assoc_null_ex(_zv, ZEND_STRL(_key));
 #define ADD_ASSOC_BOOL_EX(_zv, _key, _value) add_assoc_bool_ex(_zv, ZEND_STRL(_key), _value);
+#define ADD_ASSOC_INT64_COMPAT(_zv, _key, _value)                    \
```

Comment:
Since this forces the `int64_t` to a string (`is_bson=false`), I'd suggest we name this `ADD_ASSOC_INT64_AS_STRING`.